torch/csrc/jit/passes/onnx/peephole.cpp

```diff
@@ -59,9 +59,9 @@ bool isBroadcasting(Node *node) {
 // Note that this is NOT equivalent to numpy broadcasting semantics, and do
 // not represent that generalized broadcasting that Pytorch implements in
 // general. Rather, this is Caffe2-style broadcasting.
-bool fusibleExpandTo(at::IntList from, at::IntList to) {
+std::tuple<bool, ssize_t> fusibleExpandTo(at::IntList from, at::IntList to) {
```

Comment:
Need a doc what the two elements of the tuple mean.

Reply:
Also, we have `at::optional` now, consider using that instead of using -1 (and a signed size type) to represent no expansion.

Reply:
And maybe the docs above need updating?